### PR TITLE
DDF-1870 added documentation for users.attributes file and other areas of security

### PR DESCRIPTION
**Pull request by stustison**
@codice/security @ricklarsen

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/securing-contents.adoc [hunk 1]
```diff
@@ -1408,6 +1408,51 @@ Configuring a Java Keystore for Secure Communications
 Additional Pax-Web SSL configuration info: http://team.ops4j.org/wiki/display/paxweb/SSL+Configuration[SSL Configuration]
 ====
 
+==== Updating System Users
+
+By default, all system users are located in the `<DDF_INSTALL_DIR>/etc/users.properties` and `<DDF_INSTALL_DIR>/etc/users.attributes` files.
+The default users included in these two files are "admin" and "localhost".
+The users.properties file contains username, password, and role information; while the users.attributes file is used to mix in additional attributes.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/securing-contents.adoc]: can we monospace `users.properties` to set it apart as a filename?

Comment (stustison) [inline distribution/docs/src/main/resources/_contents/securing-contents.adoc]: Sure, what is the syntax for that?

**Comment by ricklarsen** [comment]
:+1:

File: distribution/docs/src/main/resources/_contents/securing-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/securing-contents.adoc]: surround with backticks

**Pull request merged**